src/docs/user/references/Notification-service.adoc

```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
 If a notification method parameter is deleted or changed and reused in the future, then a new confirmation
 request will be triggered.
 
+=== Channels
+
+==== Principle
+
+A notification <<_glossary_notification_channel, Channel>> is a centralized space in the <<_glossary_notification_portal, Notification Portal>>
+for groups of users to receive notifications created from a workflow using the Channel_Notification task.
+It consists of a unique name and a set of user groups authorized to subscribe to the channel.
+
+In order to receive channel notifications it is required to create one or more channels and update the channel
+<<_glossary_notification_subscription, Subscription>> to specify which channels we wish to be notified of.
+
+[title="Channel subscription"]
+image::channel-sub.png[align="center"]
+
+==== Rules
+
+- Unlike other notifications, channel notifications are retroactive. It means that when a user subscribes to a channel that already had notifications
+, they will appear to the user.
+- Channel notifications cannot be deleted, the link:../admin/ProActiveAdminGuide.html#_notification_housekeeping[Housekeeping]
+takes care of it once their life span is exceeded.
+- Notification service administrators can subscribe to any channels. Other users must be part of at least one of the authorized groups
+defined in the channel in order to subscribe to a channel, unless all groups are authorized.
+
+==== Channel Creation
+
+All users are able to create channels for any group. It is needed to provide a unique channel name and a list of groups
+which will be able to subscribe the the channel.
+
+It can either be all groups authorized, one or more groups of the current users, or any free input.
+
+[title="Channel creation"]
+image::create-channel.png[align="center"]
+
+==== Channel Management
+
+Channel management is where users can update channels by modifying a channel's name, add or remove authorized groups or delete it.
+
+A notification service administrator is able to update any channels, regardless if he is the creator of the channel or not.
+Other users can update only channels that they created.
+
+To update a channel's name or authorized groups, simply make the update and click on the "Save" button.
```

Comment:
make a single way of highlighting keywords like update, save, etc. Something like all bold or italic or whatever